### PR TITLE
SB006 - Assign Delivery Emp to Project

### PR DESCRIPTION
**Pull request by amchugh689**
task six: as a member of staff at soniak bew i want to assing a delivery employee to a project or something

this allows a user to ssign a delivery employee to a project and automatically sets their working status to true by default

### CODE REVIEW

**Review by CDawson8 — APPROVED** [review]
Your honour, I deem this pull request.... APPROVED!

**Review by jkelso662 — APPROVED** [review]
👍

File: src/main/java/org/kainos/ea/db/DeliveryProjectDao.java [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+public class DeliveryProjectDao {
+
+    private static Connection conn;
+    DatabaseConnector databaseConnector = new DatabaseConnector();
+
+    public void createDeliveryProject(DeliveryProjectRequest dp) throws SQLException {
+        Connection c = databaseConnector.getConnection();
+
+        String insertStatement = "INSERT INTO Project_Delivery(Project_ID, Delivery_ID, Actively_Working) " +
+                "VALUES(?, ?, 1)";
```
Comment (jkelso662) [inline src/main/java/org/kainos/ea/db/DeliveryProjectDao.java:20]: Looks like you ignored the value in the Actively_Working parameter! This normally wouldn't make it into main!

File: src/main/java/org/kainos/ea/resources/DeliveryProjectController.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+import javax.ws.rs.core.Response;
+
+@Api("Soniak Bew Project Delivery Link API")
+@Path("/api")
+public class DeliveryProjectController {
+    DeliveryProjectService deliveryProjectService = new DeliveryProjectService();
+
+
+    @POST
+    @Path("/projectdelivery/{id1}_{id2}")
```
Comment (jkelso662) [inline src/main/java/org/kainos/ea/resources/DeliveryProjectController.java:24]: Odd! haha

File: src/main/java/org/kainos/ea/db/DeliveryProjectDao.java [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+        st.setInt(1, dp.getProject_ID());
+        st.setInt(2, dp.getDelivery_ID());
+
+        st.executeUpdate();
+
+
+
+
+
```
Comment (jkelso662) [inline src/main/java/org/kainos/ea/db/DeliveryProjectDao.java:31]: Lots of blank space!

**Review by jkelso662 — APPROVED** [review]
A couple of small things I have pointed out but am allowing a MR for your presentation. Typically these wouldn't be allowed to be merged in this form.

**Pull request merged**